pacientes/models.py

```diff
@@ -113,6 +113,7 @@ class Paciente(Persona):
         max_length=20, null=True, choices=grupos_sanguineos
     )
     observaciones = models.TextField(blank=True, null=True)
+    ultima_actualizacion = models.DateField(default=datetime.datetime.today())
```

Comment:
Acá hay que cambiar el valor porque cada vez que ejecutas `makemigrations`, ese campo se calcula con el valor de `today()` y siempre te va a generar migración diferente.

```suggestion
    ultima_actualizacion = models.DateField(default=date.today)
```

Referencia: https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.DateField.auto_now_add